### DISCUSSION TITLE
cannot find `i3-swap-focus` as in regolith 2.1

### DISCUSSION THREAD

**Discussion by twitu** [regolith-linux/regolith-desktop#720] [Q&A] [answered]
I've updated to regolith 2.1 but I cannot find `i3-swap-focus`. [Configuration documentation](https://regolith-desktop.com/docs/using-regolith/configuration/#all-i3-configuration-packages) and [Release](https://regolith-desktop.com/docs/reference/Releases/regolith-2.1-release-notes/#new-features) mentions it's available as part of the installation but I can't find it in the config directory.

```
ls /usr/share/regolith/i3/config.d/
15_base_launchers  35_i3-gaps         40_workspace-config     60_config_keybindings  82_rofication-ilia  88_network-manager
20_ilia            40_default-style   50_resize-mode          70_bar                 84_ftue             90_user-programs
30_navigation      40_next-workspace  55_session_keybindings  80_compositor          86_unclutter
```

```
cat /etc/regolith/version
REGOLITH_VERSION=2.1
```

I wanted to check if I've missed anything obvious before filing an issue.

**Comment by kgilmer** [comment, marked as answer]
Hi @twitu , yes unfortunately this package didn't make it in as expected due to me missing including it during the release build.  It will be available in the next release (2.2), or you can add the `testing` package repository to get it now.  For example if you're on Ubuntu Jammy you can add the `testing` repository by adding this line to your `/etc/apt/sources.list.d/regolith.list`:

```shell
deb [arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] https://regolith-desktop.org/testing-ubuntu-jammy-amd64 jammy main
```

**Reply by twitu** [reply]
I haven't switched to 22.04 yet 😄. I'll wait for the 2.2 release. Thanks for the help 👍
